src/resources/postcss/tribe-common-admin/tec_baseline_admin/_header.pcss

```
@@ -25,29 +27,17 @@
 }
 
 .tec-admin-page__logo {
-	margin: 0;
-	padding: var(--tec-spacer-1);
-	vertical-align: middle;
-	width: var(--tec-spacer-7);
+	height: var(--tec-spacer-8);
+	margin-left: var(--tec-spacer-0);
+	padding: var(--tec-spacer-0);
+	width: var(--tec-spacer-8);
```

Don't use spacer vars for dimensions - they are just for whitespace.
```suggestion
	width: 40px;
```